### PR TITLE
hardening/1182_deprecate_gr_doc

### PR DESCRIPTION
**Pull request by frbattid**
* Partially implements issue #1182

### CODE REVIEW

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md [hunk 1]
```diff
@@ -79,105 +84,141 @@ Let's assume these rules:
...
+	      entityType=room,
+	      attributes=[
+	         ...
+	      ]
+	   }
+	}
+}
+```
+
+As can be seen, two entities (`suite.12` and `other.9`) of the same type (`room`) within the same FIWARE service (`hotel`) but different service paths (`/suites` and `/other`) are notified. `NGSIRestHandler` will crate two `NGSIEvent`'s:
```
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md]: crate -> create

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_name_mappings_interceptor.md [hunk 2]
```diff
@@ -92,64 +98,169 @@ Other interceptors may add further headers, such as the `timestamp` header added
...
+	      entityType=room,
+	      attributes=[
+	         ...
+	      ]
+	   }
+	}
+}
+```
+
+As can be seen, two entities (`suite.12` and `other.9`) of the same type (`room`) within the same FIWARE service (`hotel`) but different service paths (`/suites` and `/other`) are notified. `NGSIRestHandler` will crate two `NGSIEvent`'s:
```
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_name_mappings_interceptor.md]: crate again :) -> create

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md [hunk 3]
```diff
@@ -2,126 +2,112 @@
...
     * [Configuration](#section2.1)
     * [Accepted character set](#section2.2)
 * [Programmers guide](#section3)
     * [`NGSIRestHandler` class](#section3.1)
 
 ##<a name="section1"></a>Functionality
-###<a name="section1.1"></a>Mapping NGSI events to flume events
-This section explains how a notified NGSI event (an http message) containing context data is converted into a Flume event (an object in memory or a file), suitable for being consumed by any of the Cygnus sinks, thanks to `NGSIRestHandler`.
+###<a name="section1.1"></a>Mapping NGSI events to `NGSIEvent` objects
+This section explains how a notified NGSI event (a http message containing headers and payloa) is used to crate a `NGSIEvent` object, suitable for being consumed by any of the Cygnus sinks, thanks to `NGSIRestHandler`.
```
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md]: payloa -> payload 
crate -> create

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md [hunk 4]
```diff
@@ -2,126 +2,112 @@
...
-| http message | Flume event object |
+As can be seen, there is a quasi-direct translation among http messages and `NGSIEvent` objects:
+
+| http message | `NGSIEvent` object |
 |---|---|
-| `Content-Type` header | `content-type` header (discarded from 0.14.0) |
 | `Fiware-Service` header | `fiware-service` header |
 | `Fiware-ServicePath` header | `fiware-servicepath` header |
-| `Fiware-Correlator` header | `fiware-correlator` header |
+| `Fiware-Correlator` header | `fiware-correlator` header. If this header is not sent, the `fiware-correlator` is equals to the `transcation-id` header. |
```
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md]: transcation -> transaction

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md [hunk 5]
```diff
@@ -2,126 +2,112 @@
...
+	      entityType=room,
+	      attributes=[
+	         ...
+	      ]
+	   }
+	}
+}
+```
+
+As can be seen, two entities (`suite.12` and `other.9`) of the same type (`room`) within the same FIWARE service (`hotel`) but different service paths (`/suites` and `/other`) are notified. `NGSIRestHandler` will crate two `NGSIEvent`'s:
```
Comment (pcoello25) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md]: crate :)

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md (hunk 1 above)
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1308/commits/7dbe4ab1ef9d2cc343af43c0e0fe9c6a960dc4af

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_name_mappings_interceptor.md (hunk 2 above)
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_name_mappings_interceptor.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1308/commits/7dbe4ab1ef9d2cc343af43c0e0fe9c6a960dc4af

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md (hunk 3 above)
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1308/commits/7dbe4ab1ef9d2cc343af43c0e0fe9c6a960dc4af

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md (hunk 4 above)
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1308/commits/7dbe4ab1ef9d2cc343af43c0e0fe9c6a960dc4af

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md (hunk 5 above)
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1308/commits/7dbe4ab1ef9d2cc343af43c0e0fe9c6a960dc4af

**Pull request merged**